.circleci/config.yml

```diff
@@ -47,16 +47,196 @@ jobs:
             - "/usr/local/bin"
             - "/usr/local/lib/python3.7/site-packages"
       - run:
+          name: Make directory for test results
           command: |
             mkdir /tmp/test-results;
       - run:
-          # Run unit tests
+          name: Run unit tests
           command: |
             pipenv run crt_portal/manage.py test cts_forms
+      - run:
+          name: Run bandit (Python security) tests
+          command: |
+            pipenv run bandit -r crt_portal/
+# # we are not ready for this
+#       - run:
+#           name:
+#           command: |
+#             pipenv run flake8
+
       - store_test_results: # Upload test results for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
           path: /tmp/test-results
+
+  deploy-dev:
+    working_directory: ~/code
+    docker:
+      # CircleCI Python images available at: https://hub.docker.com/r/circleci/python/
+      - image: python:3.7
+        environment: # environment variables for primary container
+          PIPENV_VENV_IN_PROJECT: true
+          DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
+      # CircleCI PostgreSQL images available at: https://hub.docker.com/r/circleci/postgres/
+      - image: postgres:10.1-alpine
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: circle_test
+    steps:
+      - checkout
+      - restore_cache:
+          # this should all be cached from the test
+          key: deps9-{{ .Branch }}-{{ checksum "Pipfile.lock" }}-{{ checksum "package.json" }}
+      - run:
+          name: Node install
+          command: |
+            echo "deb https://deb.nodesource.com/node_10.x stretch main" > /etc/apt/sources.list.d/nodesource.list && \
```

Comment:
Hopefully in a later PR we can D.R.Y. up some of this repeated Node-installing code by pulling from our Docker setup ... this CircleCI tutorial may be helpful! https://circleci.com/blog/how-docker-helps-you-keep-your-ci-dry/

That said, this looks great for now.